mypy/nodes.py

```diff
@@ -2439,26 +2439,33 @@ class TypeVarLikeExpr(SymbolNode, Expression):
     Note that they are constructed by the semantic analyzer.
     """
 
-    __slots__ = ("_name", "_fullname", "upper_bound", "variance")
+    __slots__ = ("_name", "_fullname", "upper_bound", "default", "variance")
 
     _name: str
     _fullname: str
     # Upper bound: only subtypes of upper_bound are valid as values. By default
     # this is 'object', meaning no restriction.
     upper_bound: mypy.types.Type
+    default: mypy.types.Type
```

Comment:
Add a comment here, as with the other fields. Feel free to tweak the wording if you prefer something else:
```suggestion
    # Type used to resolve the TypeVar if the default is not explicitly given. See PEP 696.
    default: mypy.types.Type
```